src/Stepper/StepperHeader.jsx

```diff
@@ -70,11 +71,17 @@ StepperHeader.propTypes = {
   className: PropTypes.string,
   /** A component that receives `activeStepIndex` and `totalSteps` props to display them. */
   PageCountComponent: PropTypes.elementType,
+  /** The max width in which the compact view of the header will switch to display the step number that is
+   * currently in progress. To change the max width, use 'extraSmall', 'medium', 'large',
+   * 'extraLarge', 'extraExtraLarge'
+   */
+  compactWidth: PropTypes.string,
```

Comment:
[suggestion] Perhaps use `PropTypes.oneOf(['xs', 'sm', 'md', 'lg', 'xl', 'xxl'])`. Most of the `size` related props we have utilize the shorthand properties. Also note, the default prop below of `small` isn't listed in this comment.